DEV-3/DEV-3.sln

```diff
@@ -0,0 +1,25 @@
+﻿
```

Comment:
It's better to have one solution for all homeworks, not urgent but try to do this starting from task 4